goblint.opam

```diff
@@ -20,7 +20,7 @@ doc: "https://goblint.readthedocs.io/en/latest/"
 bug-reports: "https://github.com/goblint/analyzer/issues"
 depends: [
   "ocaml" {>= "4.10"}
-  "dune" {>= "3.0"}
+  "dune" {>= "2.9" & >= "3.0"}
```

Comment:
What is the reason for this change?

Reply:
> What is the reason for this change?

I presume this might have been added as a constraint by mistake while one of us in the team ran `opam` locally.

Reply:
This should be now resolved by https://github.com/goblint/analyzer/pull/928/commits/1f9674dc07059d9715f9746b29dff04c9e81fa72